src/rendering/renderers/shared/shader/utils/createUBOElements.ts

```diff
@@ -1,7 +1,13 @@
 /* eslint-disable quote-props */
-export type UNIFORM_TYPES =
+export type UNIFORM_TYPES_SINGLE =
```

Comment:
is there a need to export this?
I think we would want everyone to use just `UNIFORM_TYPES`?